### PR TITLE
Test "Add document url for LSP"

### PR DESCRIPTION
**Pull request by 928PJY**

### CODE REVIEW

**Review by mingwli — APPROVED** [review]

File: docs/specs/lsp.yml [hunk 1]
```diff
@@ -1,7 +1,9 @@
 ---
 # Basic lsp test
 inputs:
-  docfx.yml:
+  docfx.yml: |
```
Comment (mingwli) [inline docs/specs/lsp.yml:4]: ????

**Comment by yufeih** [comment]
Finally 🎉 🎉 🎉 🎉 🎉 🎉 🎉

**Pull request closed without merging**